build/scripts/DownloadDotNetRuntimeInstaller.ps1

```diff
@@ -0,0 +1,43 @@
+[CmdLetBinding()]
+Param(
+    [string]$Platform = "x64",
```

Comment:
Why default `$Platform`? Why not always require it (no default)?

This comment applies throughout

Reply:
Yeah, I'm updating this to mandatory